salloc, sattach, sbcast, sdiag, sprio, sshare, strigger, sview, sh5util, scrun, scrontab: add pages (#11651)
tldr-pages/tldr: pages/linux/sattach.md

# sattach

> Attach to a Slurm job step.
> More information: <https://slurm.schedmd.com/sattach.html>.

> - Make available the IO streams (`stdout`, `stderr`, and `stdin`) of a Slurm job step:

Reviewer: Define `Make available` here.

Author: ```suggestion
-  Redirect the IO streams (`stdout`, `stderr`, and `stdin`) of a Slurm job step to the current terminal:
```

Do you think this would be better?

Reviewer: Yes, that looks much better @gutjuri. Many thanks!

`sattach {{jobid}}.{{stepid}}`

> - Transmit `stdin` to the specified task only:

Author: ```suggestion
- Use the current console's input as `stdin` to the specified task:
```

Not sure how to formulate this better.

Reviewer: That suggestion reads much better - let's do that.

> `sattach --input-filter {{task_number}}`

Reviewer: What do you mean by `task number`? Is this a job number, or something else?

Try to use the official language from the project itself.

Further, what do you mean by transmit to the specified task? I'm rather confused. Is this attaching interactively, or pointing something else at a job's stdin?

Author: "Task" is an official term. I cannot find a document providing an exact definition, but the term is mentioned several times in the docs, e.g. here https://slurm.schedmd.com/cpu_management.html#Section2

See also: https://stackoverflow.com/questions/39186698/what-does-the-ntasks-or-n-tasks-does-in-slurm

Reviewer: Hmm okay, that makes sense. Thanks for the clarification.

Could you explain what you mean by `transmit to the specified task` though please?

Author: Basically that I send input from the current console to the specified task.

> - Only display `stdin`/`stderr` of the specified task:

Reviewer: Please try to keep the language consistent. In this page alone you've used `display`, `transmit`, and `make available`. This is very confusing.

Author: ```suggestion
- Only redirect `stdin`/`stderr` of the specified task:
```
